content/broker/broker-transition/technical-overview/_index.en.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Technical Overview
+linktitle: Technical Overview
+description: The Altinn 3 Broker Transition Solution allows service owners who own broker services in Altinn 2 to point those services towards Altinn 3 using an internal Altinn 2 bridge.
+tags: [architecture, solution, broker, transition]
+toc: false
+weight: 1
+---
+
+## About
+
+The Altinn 3 Broker Transition service bridge is an internal component in Altinn 2 that transfers Broker requests from Altinn 2 to Altinn 3 for a given request, based on the Service/ServiceCode combination of the request.
+This is an implementation of the soft shift solution described [here](../../solution-architecture/#soft-shift-from-altinn-2-to-altinn-3).
+
+## Technical overview
+
+Altinn 2 allows end users in Altinn 2 to make BrokerService requests for specific Broker Services that will be transferred to Altinn 3 instead of being stored in Altinn 2.
+Files transferred in this way will be available for Altinn 3 users and Altinn 2 users.
+
+
+1. Requests to that have the ability to specify ServiceCode / ServiceEdition. 
```

Comment:
change to "Requests that have the ability..."

Reply:
done